elements/lisk-validator/src/keywords/data_type.ts

```diff
@@ -60,19 +69,27 @@ const compile = (
 	}
 
 	return (
-		_data: string,
+		_data: Buffer | bigint | string | number,
```

Comment:
The name `_data` was used to by pass the limiting error of unused variable. You can rename it back to `data`.

```suggestion
		data: Buffer | bigint | string | number,
```